libs/langchain/langchain/vectorstores/supabase.py

```diff
@@ -88,6 +89,7 @@ def __init__(
         client: supabase.client.Client,
         embedding: Embeddings,
         table_name: str,
+        chunk_size: int,
```

Comment:
can we give this a default value of 500? otherwise this is a breaking change

Reply:
I believe it was handled at 108 but shouldn't hurt anything to have it everywhere. Pushed
https://github.com/langchain-ai/langchain/blob/9bbbdc35878d9f328e616c08fbb51cfcc7310e66/libs/langchain/langchain/vectorstores/supabase.py#L108
I tested with chunk_size not being defined in the .from_documents call and it defaulted properly.